azkaban-web-server/src/main/java/azkaban/webapp/servlet/HistoryServlet.java

```diff
@@ -91,7 +88,7 @@ private void handleHistoryPage(final HttpServletRequest req,
             "azkaban/webapp/servlet/velocity/historypage.vm");
     int pageNum = getIntParam(req, "page", 1);
     final int pageSize = getIntParam(req, "size", 16);
-    page.add("vmutils", this.vmHelper);
+    page.add("vmutils", new ServletUtil(this.projectManager));
```

Comment:
The key name and `ServletUtil` class name doesn't match perfectly. I would believe `VmUtils` class name would be appropriate in this case, since this helper class is not used in a Servlet.

Reply:
renamed to VelocityUtil